tests/sequential/prometheus_metrics/prometheus_metrics_test.go

```diff
@@ -807,3 +853,36 @@ func checkMetricServerValues(t *testing.T, families map[string]*prommodel.Metric
 	}
 	assert.GreaterOrEqual(t, metricValue, 1.0, "apiserver_request_total has to be greater than 0")
 }
+
+func testPauseMetric(t *testing.T, data templateData) {
```

Comment:
test here following:

1. have running ScaledObject -> check that paused metric is set to false
2. pause the SO -> check that  paused metric is set to true
3. unpause the SO -> check that paused metric is set to false